app.js

```diff
@@ -198,16 +199,19 @@ function questions(){
       console.log('trySeven set to ' + trySeven);
     }
   }while(!correctAnswerSeven && trySeven < 6);
+}//end of quiz questions
+
+function score (){
+  if(numCorrect === 0){
+    alert('You got ' + numCorrect + ' out of 7...  I\'m an interesting person ' + userName + ', you should learn about me!');
```

Comment:
Consider how you would reference the user's name here if you took my suggestion earlier to avoid assigning to a global variable.